unit-tests/sw-dev/test-sw-dev-motion.py

```diff
@@ -0,0 +1,58 @@
+# License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Once you are in sw-dev folder no need for `sw-dev` in the name,
It makes the test name be:

![image](https://github.com/IntelRealSense/librealsense/assets/64067618/0b9bda59-c78a-4f9a-a71c-0f3700fbe441)

Reply:
I meant above that the final name will be test-sw-dev-motion :)
Sorry if I mislead you.
You should rename to `test-motion.py` and verify on GHA that the output test name is `test-sw-dev-motion'